### ISSUE TITLE
Seems like the grid metrics are wrong for geostrophic shear calculation

### ISSUE THREAD

**Issue by cspencerjones** [raphaeldussin/MOM6-AnalysisCookbook#19]
I'm talking about cell number 12 of https://mom6-analysiscookbook.readthedocs.io/en/latest/notebooks/Buoyancy_Geostrophic_shear.html . Because the derivatives here are part of a gradient operator (not div or curl), it isn't necessary to weight by `ds.dyt` and `ds.dxt`. 

I don't think I can do a PR without the underlying data, and I would like someone to sanity check me if possible. One of my group members is using this cookbook to start working on some output. They are generally a great resource so thank you @raphaeldussin !!

**Comment by raphaeldussin** [comment]
@cspencerjones are you suggesting to replace with `ddens_dx = grid.diff(anom_density, 'X', boundary='fill') / ds.dxCu` ?
@hmkhatri any thoughts on this?

**Comment by hmkhatri** [comment]
Agree with @cspencerjones 
@raphaeldussin I think It should be 

```
ddens_dx = grid.diff(anom_density, 'X', boundary='fill') / ds.dxCu
ddens_dy = grid.diff(anom_density, 'Y', boundary='fill') / ds.dyCv
```

**Comment by raphaeldussin** [comment]
closed by #20
